NotifyMe/models/notification.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from django.utils import timezone
+from datetime import timedelta
+from .notificationType import NotificationType
+
+#-------NOTIFICATION MODEL---------
+class Notification(models.Model):
+  title = models.CharField(max_length=40)
```

Comment:
all this values should come from setting.py e.g. declare NOITIFICATION_NAME_LENGTH=60
```suggestion
  title = models.CharField(max_length=40)
```